tee-worker/service/src/main.rs

```diff
@@ -193,8 +193,10 @@ fn main() {
 
 	#[cfg(feature = "mockserver")]
 	thread::spawn(move || {
-		info!("*** Starting mock server");
-		lc_mock_server::run();
+		if !config.disable_mock_server {
+			info!("*** Starting mock server");
+			lc_mock_server::run();
+		}
```

Comment:
May I ask why we need `disable_mock_server`? It's guarded by `mockserver` feature and in CI we use the `sidechain` image for testing resume-worker

Reply:
I usually use `mockserver` for local compilation. So if I start 2 workers locally, I need this option to disable starting mockserver.